methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    if not Use_Staged_Data:
</pre><p>@<a href="https://github.com/sssangha" target="_blank">sssangha</a> I think we would not list that here. And let Mintpy decide to not re-run things.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>